src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+@import 'header';
+@import 'variables';
+@import 'card';
+@import 'stars';
+@import 'catalog';
```

Comment:
Imports should be placed at the top of the document before any other rulesets according to the code style rules. This helps with maintaining a clear hierarchy and readability of the code.